.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,15 +2,44 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
 **Environment**
 - Loader: [Forge/Fabric]
+- Loader Version:[]  <!-- Do not put 'latest' as a  version that is not valid  -->
```

Comment:
sometimes they might us an outdated version of the loader that a patch might have already  been fixed in the newer version of the loader